backend/qemu.pm

```diff
@@ -610,6 +610,9 @@ sub start_qemu ($self) {
         }
     }
 
+    # fallback check for RHEL varients qemu-kvm
+    $qemubin = find_bin('/usr/libexec/', qw(kvm qemu-kvm)) unless $qemubin;
```

Comment:
Maybe this change could be split into another commit (not PR). At least it doesn't seem to be strictly related to s390x.